assets/src/apps/authoring/components/PropertyEditor/schemas/lesson.ts

```diff
@@ -216,7 +216,7 @@ export const transformSchemaToModel = (schema: any) => {
     additionalStylesheets,
     title: schema.Properties.title,
     customCss: schema.Properties.customCSS || '',
-    customScript: schema.CustomLogic.customScript,
+    customScript: schema.CustomLogic.customScript || '',
```

Comment:
There was a bug where you couldn't completely erase the customScript, this fixes that.